src/overlays/actors/ovl_En_Kakasi3/z_en_kakasi3.c

```diff
@@ -21,35 +47,378 @@ const ActorInit En_Kakasi3_InitVars = {
     (ActorFunc)EnKakasi3_Update,
     (ActorFunc)EnKakasi3_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Kakasi3/EnKakasi3_Destroy.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Kakasi3/EnKakasi3_Init.s")
+void EnKakasi3_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnKakasi3* this = THIS;
+
+    Collider_DestroyCylinder(globalCtx, &this->collider);
+}
+
+void EnKakasi3_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnKakasi3* this = THIS;
+
+    osSyncPrintf("\n\n");
+    // Translates to: Obonur -- Related to the name of the scarecrow (Bonooru)
+    osSyncPrintf(VT_FGCOL(YELLOW) "☆☆☆☆☆ おーボヌール ☆☆☆☆☆ \n" VT_RST);
+    this->actor.unk_1F = 6;
+
+    Collider_InitCylinder(globalCtx, &this->collider);
+    Collider_SetCylinder(globalCtx, &this->collider, &this->actor, &sCylinderInit);
+    SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_060065B0, &D_06000214, NULL, NULL, 0);
+    this->actor.flags |= 0x400;
+    this->rot = this->actor.posRot.rot;
+    this->actor.colChkInfo.mass = 0xFF;
+    Actor_SetScale(&this->actor, 0.01f);
+    this->actionFunc = func_80A911F0;
+}
+
+void func_80A90E28(EnKakasi3* this) {
+    this->unk_1AE = 0;
+    this->unk_1A4 = 0;
+    this->skelAnime.animPlaybackSpeed = 0.0f;
+    this->unk_1AA = this->unk_1AE;
```

Comment:
Does something like this work by any chance? Might have to reorder things to keep it matching
```suggestion
    this->unk_1A4 = 0;
    this->skelAnime.animPlaybackSpeed = 0.0f;
    this->unk_1AA = this->unk_1AE = 0;
```